management_system/tests/test_views/test_dish_view.py

```diff
@@ -0,0 +1,43 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+from management_system.models import DishType, Dish
+
+
+class PrivateDishPageTestCase(TestCase):
+    def setUp(self):
```

Comment:
if you were using function based tests, you could create authenticated_user fixture once and user it for all the testsm instead of writing the same presetup

Reply:
Thank you, I will try this approach of writing tests for my next project.